### DISCUSSION TITLE
NZP HUD,SFX,MUSIC Overhaul

### DISCUSSION THREAD

**Discussion by slimshayneeYT** [nzp-team/nzportable#900] [Other Content Releases]
Hello yall! My name is slimshaynee, here to bring you this mod I made for NZ:P. This mod is called The NZ:P OG Experience. This mod changes sounds for weapons, movement, zombie, announcer, machines, and more. This mod also features a new menu, HUD, FONT, Skybox, muzzle flash, hit marker, and a couple more FX in game. Lastly, I added some music in the menu and on a couple maps with some OG zombies music. Please support the original NZ:P Dev team, as they put so much effort into this game over the years. This mod is just another way to play this awesome game. Right now, the mod is available on PC and PSVita; more consoles soon! Here is a link to my YouTube video showcasing some of the changes. File size is larger than 25MB so I have the mod for download on Google Drive.

Video: https://youtu.be/120kn58fTbY?si=pwPrx2rha4hHA4v3

Download from Google Drive: https://drive.google.com/drive/folders/1-JshfoSLUTT6r0RP5fy8y5x6kur-4SGF

![NZPOGEWide](https://github.com/user-attachments/assets/5bed8a6c-7241-4e80-8239-1abb5f20594d)
![1](https://github.com/user-attachments/assets/113f3a47-7259-423b-af5a-c009cb2c3182)
![2](https://github.com/user-attachments/assets/8476c72f-0292-4501-97d8-7eb4e892e516)
![3](https://github.com/user-attachments/assets/adf27810-75b7-4743-b0b9-3657ee0c683f)
![4](https://github.com/user-attachments/assets/3e7c21c8-1999-4418-90ee-494399a82d14)
![5](https://github.com/user-attachments/assets/71bda15e-d03f-433e-999f-f4384c82df5b)
![6](https://github.com/user-attachments/assets/8af1c247-7870-44ad-a081-73eac4d15f6e)
![7](https://github.com/user-attachments/assets/ff3919df-33aa-4040-ab96-753a936a4280)

**Comment by Zambales7** [comment]
I try downloading it but for some reason download button ain't appearing in google drive

**Reply by lucasgvilhena10-ai** [reply]
Yes really man

**Comment by slimshayneeYT** [comment]
My sincere apologies! I will make sure to fix it when I get back home from
work. :)

**Comment by slimshayneeYT** [comment]
Should be good to download! Let me know if you have any issues. :)

**Comment by Naievil** [comment]
I like the passion!

**Reply by slimshayneeYT** [reply]
Thank you so much! I really appreciate your support! :)

**Comment by EngineerGaming123** [comment]
where exactly would I put the files in their respective spots for this to work?
